data/tutorials/CAT23001/default-physics-model-datacard.txt

```diff
@@ -0,0 +1,15 @@
+imax 1  number of channels
+jmax 1  number of backgrounds
+kmax 2  number of nuisance parameters (sources of systematical uncertainties)
+------------
+# One channel, in which we observe 0 events
+bin            ch1
+observation    0
+# ------------
+bin            ch1    ch1
+process        S      B
+process        0      1
+rate           4.76   1.47
+# ------------
+deltaS  lnN    1.20   -      20\% uncertainty on signal
+deltaB  lnN    -      1.50   50\% uncertainty on background
```

Comment:
I wonder if its worth adding a '#' in front of the comments for these lines to make it clear they are comments (despite the fact that they run fine).